docs/docs/articles/github-actions.md

```diff
@@ -1,135 +1,85 @@
-# GitHub Actions
-
-In order to automate Testkube runs, access to a K8s cluster is needed. For example, a configured environment with the set up context and kubeconfig for communication with the K8s cluster.
-
-Testkube uses your K8s context and access settings in order to interact with the cluster and tests resources, etc.
-
-In the next few sections, we will go through the process of Testkube and Helm (for Testkube's release deploy/upgrade) automations with the usage of GitHub Actions and GKE K8s.
-
 ## Testkube GitHub Action
 
-The testkube GitHub Action is available here <https://github.com/kubeshop/testkube-docker-action> and it enables running the Testkube CLI commands in a GitHub workflow.
+The Testkube GitHub Action installs Testkube and enables running any [Testkube CLI](https://docs.testkube.io/cli/testkube) command in a GitHub workflow. It is available on Github Marketplace <https://github.com/marketplace/actions/testkube-action>.
+The action provides a flexible way to work with your pipeline and can be used with Testkube Cloud or self-hosted platform (self-hosted platform means a k8s cluster that uses an open source Testkube solution (OSS), not Cloud).
```

Comment:
Actually, that will work for self-hosted Enterprise too, so maybe worth to rephrase it

Reply:
`...and can be used with Testkube Cloud, Testkube Enterprise or self-hosted platform (self-hosted platform means a k8s cluster that uses an open source Testkube solution (OSS)).`

would this work?

Reply:
`Testkube Enterprise` is self-hosted too, so it's rather incorrect. We should rather not call `OSS` specifically a self-hosted platform, as it applies to both.

More likely something like:
> ...and can be used with both Testkube Cloud, Testkube Enterprise, and open-source Testkube installation.

But I'm not sure too